### ISSUE TITLE
NVDA should have the ability to report changes in an object

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1481]
**Reported by parham on 2011-05-03 07:41**
It happens a lot of times that we need to monitor an object for changes (E.G. a status bar, or something as simple as "copying new files..."). NVDA should allow the user to monitor an object for changes, and it should speak the new, changed text.
**Blocked by #498**

**Comment by bhavyashah** [comment]
@jcsteh At one point in time, you set the milestone of this ticket to near-term, and the proposed ability does seem quite useful to me in a multiplicity of usage scenarios. Could you please clarify your current stand on this ticket and assign this a priority as well?

**Comment by Adriani90** [comment]
@feerrenrut, @michaelDCurran any progress on this? AS stated in ä498 part of necessary code i written in the add-on golden cursor which is maintained by @josephsl. But I guess the best time to implement this would be after implementation of speech refactoring, quoing and so on. So I guess blocked by #7599. Right?

**Comment by Adriani90** [comment]
Actually, this is a duplicate of #498. Closing as duplicate.
